src/Core/Scheduler/Scheduler.js

```diff
@@ -130,6 +131,7 @@ Scheduler.prototype.initDefaultProviders = function initDefaultProviders() {
     this.addProtocolProvider('tile', TileProvider);
     this.addProtocolProvider('3d-tiles', $3dTilesProvider);
     this.addProtocolProvider('potreeconverter', PointCloudProvider);
+    this.addProtocolProvider('orientedImage', OrientedImageProvider);
```

Comment:
You have to add a new provider, you can not use the `dataSourceProvider`?

Reply:
No, I can't use dataSourceProvider, because the urls I need to build doesn't depend on extent.